PowerShellEditorServices.build.ps1

```diff
@@ -223,24 +225,31 @@ function UploadTestLogs {
     }
 }
 
+function XunitTraitFilter {
+    $TraitFilter = ''
+    # Reference https://docs.microsoft.com/en-us/dotnet/core/testing/selective-unit-tests
+    if (![string]::IsNullOrEmpty($TestFilter)) { $TraitFilter = "-trait $TestFilter" }
```

Comment:
Since `$TestFilter` is type string, it is sufficient to use the more canonical `if (!$TestFilter) { ... }`

Reply:
Actually this whole function could be simplified to just:
```
if ($TestFilter) { "-train $TestFilter" } else { "" }
```

Reply:
```powershell
if (!$TestFilter)
{
    return ""
}

"-train $TestFilter"
```

Reply:
I can change it but I'm not really a fan of that kind of operation. It makes things like `if ("False") { Write-Out "This is True" }` confusing.  I prefer explicit checks

Reply:
`if ($TestFilter)` is still an explicit check of "is string null or empty" and sorry but `"False"` is not null or empty.  I mean,  underneath PS is just doing string.IsNullOrEmpty for you.

Reply:
Also, in the vein of please follow the coding style that already exists in the script, you will see this style:
```
if (-not $name)
...
if ($env:APPVEYOR) {
```

Reply:
Changed.

Reply:
My feeling is that it's a common convention in Python, JS and PowerShell for a reason. It can be abused, but this case is precisely the design intent. Same as `return value || default`.

In PowerShell's case, calling the C# APIs is sometimes less robust; PowerShell's coercions absorb more possible bad cases with less code.